src/backend/nodes/readfuncs.c

```diff
@@ -32,12 +32,14 @@
 
 #include <math.h>
 
+#include "catalog/pg_collation.h"
 #include "fmgr.h"
 #include "miscadmin.h"
 #include "nodes/extensible.h"
 #include "nodes/parsenodes.h"
 #include "nodes/plannodes.h"
 #include "nodes/readfuncs.h"
+#include "parser/parser.h"	/* For sql_dialect */
```

Comment:
Why do we need this?

Reply:
It's not required. Removed in new revision.